src/Plugin/views/display/GraphQL.php

```diff
@@ -94,9 +95,22 @@ protected function defineOptions() {
     unset($options['exposed_block']);
     unset($options['css_class']);
 
+    $options['graphql_query_name'] = ['default' => ''];
     return $options;
   }
 
+  public function getGraphQLQueryName() {
+    $graphql_query_name = strip_tags($this->getOption('graphql_query_name'));
+    if (empty($graphql_query_name)) {
+      $viewId = $this->view->id();
+      $displayId = $this->display['id'];
+      // @todo: fix PHP fatal when graphql_core is not installed.
+      // fix: move this GraphQL display plugin under graphql_views module.
```

Comment:
We can't move it there because this plugin can also be used with custom, n on generated schemas. And it should stay that way.